### PR TITLE
chore: add homepage and repo directory to `package.json`s

### PR DESCRIPTION
**Pull request by maximepvrt**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by maximepvrt** [comment]
@danielroe why did you replace [the short syntax](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#repository) for repository in the last commit  https://github.com/nuxt/nuxt/commit/54529c17dbf0599d35ddf1f7e852c92b2370755e

**Comment by maximepvrt** [comment]
@danielroe updated

File: packages/schema/package.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
   "version": "3.8.1",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/nuxt/nuxt.git"
+    "url": "git+https://github.com/nuxt/nuxt.git",
+    "directory": "packages/schema"
   },
   "description": "Nuxt types and default configuration",
```
Comment (danielroe) [inline packages/schema/package.json:9]:
```suggestion
  "description": "Nuxt types and default configuration",
  "homepage": "https://nuxt.com/",
```

File: packages/vite/package.json [hunk 2]
```diff
@@ -3,7 +3,8 @@
   "version": "3.8.1",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/nuxt/nuxt.git"
+    "url": "git+https://github.com/nuxt/nuxt.git",
+    "directory": "packages/vite"
   },
   "description": "Vite bundler for Nuxt",
```
Comment (danielroe) [inline packages/vite/package.json:9]:
```suggestion
  "description": "Vite bundler for Nuxt",
  "homepage": "https://nuxt.com/",
```

**Review by danielroe — APPROVED** [review]

File: packages/vite/package.json [hunk 3]
```diff
@@ -3,9 +3,11 @@
...
+  "homepage": "https://nuxt.com/",
```
Comment (DamianGlowala) [inline packages/vite/package.json:10]:
```suggestion
  "homepage": "https://nuxt.com",
```

File: packages/webpack/package.json [hunk 4]
```diff
@@ -3,9 +3,11 @@
   "version": "3.8.1",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/nuxt/nuxt.git"
+    "url": "git+https://github.com/nuxt/nuxt.git",
+    "directory": "packages/webpack"
   },
   "description": "Webpack bundler for Nuxt",
+  "homepage": "https://nuxt.com/",
```
Comment (DamianGlowala) [inline packages/webpack/package.json:10]:
```suggestion
  "homepage": "https://nuxt.com",
```

File: packages/schema/package.json [hunk 5]
```diff
@@ -3,9 +3,11 @@
...
+  "homepage": "https://nuxt.com/",
```
Comment (DamianGlowala) [inline packages/schema/package.json:10]:
```suggestion
  "homepage": "https://nuxt.com",
```

**Pull request merged**